src/types.ts

```diff
@@ -1,9 +1,37 @@
 type Action = "load" | "save"
 
 type ActionDetails = {
-  action: string
+  cb_name: string
   body_args?: string[]
-  include?: string[]
+  modify?: FieldModify[]
+}
+
+type ActionData = {
+  octokit_cb: CallableFunction
+  action_details: ActionDetails
+  pattern: Record<string, any>
+}
+
+type FieldModify = {
+  field: string
+  rename?: string,
+  replace_for?: {
+    field: string,
+    from: IncludeFromEnum
+  }
+}
+
+enum IncludeFromEnum {
```

Comment:
Ah - so TS enums are bad news and should be avoided: https://www.executeprogram.com/blog/typescript-features-to-avoid
In any case, this issue will be resolved with a new refactoring - see the issues

Reply:
Good reading on Typescript enums, thanks !
Changed !